plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -1267,4 +1290,17 @@ public List<PreBuiltAnalyzerProviderFactory> getPreBuiltAnalyzerProviderFactorie
             );
         return factories;
     }
+
+    public void onStreamManagerInitialized(StreamManager streamManager) {
+        this.streamManager = streamManager;
+        mlEngine.setStreamManager(streamManager);
+        mlEngine.setThreadPool(threadPool);
+        streamManagerWrapper.setStreamManager(streamManager);
```

Comment:
Seems the `streamManagerWrapper` just for providing access of `streamManager`? Can we add `mlEngine.getStreamManager()` method ?